src/theme/theme.tsx

```diff
@@ -18,5 +17,12 @@ export const createCustomTheme = (darkMode: boolean) => {
         primary: darkMode ? "#646bf3" : "#333333",
       },
     },
+    components: {
+      MuiTypography: {
+        defaultProps: {
+          gutterBottom: true
+        },
+      },
+    },

```

Comment:
Sorry, I think you misunderstood me, I meant this as an example, but if you really need gutterbottom for all typographies you can leave it
What I meant that for example if you set the same borderRadius in each (or in most) Button in the project, this means you should place this style in Theme, so it automatically will apply to each MUIButton in the project, so you dont need to add this style in each button